packages/accessibility/src/behaviors/Carousel/carouselItemBehavior.ts

```diff
@@ -16,8 +17,13 @@ const carouselItemBehavior: Accessibility<CarouselItemProps> = props => ({
   },
 
   keyActions: {
-    root: {},
-  },
+    root: {
+      stopPropagation: {
```

Comment:
This looks like a strange name for keyAction. Can we use something related to the fact that navigation happend?